apps/eigen-widget/tokens.css

```
@@ -14,31 +21,46 @@
-  --shadow-sm: 0 2px 6px oklch(0.18 0.02 250 / 0.08);
-  --shadow-md: 0 8px 16px oklch(0.18 0.02 250 / 0.1);
-  --shadow-lg: 0 20px 40px oklch(0.18 0.02 250 / 0.14);
+  --chart-1: oklch(0.55 0.16 40);
+  --chart-2: oklch(0.74 0.14 70);
+  --chart-3: oklch(0.5 0.08 250);
+  --chart-4: oklch(0.68 0.1 140);
+  --chart-5: oklch(0.6 0.11 300);
 
-  --font-display: "IBM Plex Serif", "Georgia", serif;
-  --font-body: "IBM Plex Serif", "Georgia", serif;
+  --font-display: "IBM Plex Serif", Georgia, serif;
```

Stylelint (17.7.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)